shared/enums.proto

```diff
@@ -257,6 +257,14 @@ enum StokesType {
     Q = 2;
     U = 3;
     V = 4;
+    RR = 5;
```

Comment:
We don't currently have any descriptions in this file, but I think it would be useful to document this particular enum to note that although it's called `StokesType` for legacy reasons it actually includes other polarization types.

Reply:
Ok. I will add a description

Reply:
why not just rename it?

Reply:
Will that not impact backwards compatibility? If not, then that is the simplest solution. :)

Reply:
in terms of what gets sent in the actual protobuf message, it won't. But it would require backend/frontend changes  I guess 🤔

Reply:
I would be inclined to make a separate issue to rename Stokes to polarization everywhere. Although since we've already created three matching PRs to make this change, maybe this is a good time to do that renaming.